cyences_app_for_splunk/default/data/ui/views/cs_configuration.xml

```diff
@@ -530,6 +530,24 @@
               <button id="macro_cs_ad_password_change_outside_working_hour_filter_button">Update</button>
             </th>
           </tr>
+          <tr>
+            <th style="text-align:left">cs_ad_bulk_user_creation_deletion_count_limit</th>
```

Comment:
We are not generally adding limit related macros with filter categories. So let's remove it for now, we'll see if we need it, we'll add it in the right place.

Reply:
As discussed over chat, keeping as it is as we already have few limit macros in the configuration dashboard